package.json

```diff
@@ -16,15 +16,16 @@
     "scripts": {
         "start": "node node_modules/@storybook/react/bin/index -p 9001 -c .storybook",
         "build-storybook": "node node_modules/.bin/build-storybook -c .storybook -o ./storybook-static",
-        "test": "npm run lint && npm run stylelint && npm run test:ci",
+        "test": "npm run lint && npm run test:ci",
         "test:ci": "npm run build-ts && cp src/styles.scss lib && mocha --require lib/__tests__/setupEnzyme.js --recursive lib/**/*.test.js",
         "format": "prettier --write 'src/**/*.{ts, tsx, js,jsx,json}'",
         "pretty-quick": "pretty-quick",
         "prepare": "npm run dist",
         "build-ts": "rm -rf lib && tsc",
         "dist": "rm -rf dist && webpack --mode production --progress --display-modules --config webpack.config.js",
         "stylelint": "stylelint '**/*.scss'",
-        "lint": "tslint -c tslint.json src/__tests__/components/*.tsx src/components/**/**/*.tsx src/components/**/**/**/*.tsx src/utils/*.tsx",
+        "lintts:fix": "eslint --fix --ext=jsx,ts,tsx src",
```

Comment:
Is this a command to run manually?

Reply:
You can run it with `npm run lintts:fix` ... I have never been able to get `npm run lint --fix` to work with ESLint but maybe I've just not set it up properly?

Reply:
Yeah, I meant more of a, will it be called in other scripts or other processes such as Github Actions?

Reply:
`npm run lint` is run in `npm run test`, so it should be running in everything?  I'll double check tho.  
Is it general practice to run --fix in CI?  Or is it normal to let it fail and have the user go back and double check it?

Reply:
I'm not sure. We haven't had a `:fix` script in our process before. It makes sense to run that when we commit but not in CI since we want to see the changes in PRs and in CI it doesn't matter or the code shouldn't suddenly change (if it makes any "fixes" we can't see).